src/index.js

```diff
@@ -1,4 +1,4 @@
-/* global graphData countries countryPerformanceOnRiskViews asn tempData risks*/
+/* global graphData countries countryPerformanceOnRiskViews asn tempData risks ASPerformanceViews tempDataForASPerformance*/
 import React from 'react';
```

Comment:
@zelima this will be removed once API is fixed - having top ~9 AS data. For now I am mocking it as we don't have that data. It is for CountryPage.